site/src/components/util/extent.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: component
+title: Extent
+component: util.extent.js
+namespace: util
+
+example-code: |
```

Comment:
elsewhere we use `example-code` front-matter variables for a couple of reasons:

 1. To ensure that the code being executed is identical to the code displayed to the user
 2. So that we can re-use code, e.g. showing an example on the front page.

Neither apply in this case, so feel free to add the code snippets directly in to the body of the post.

I'd also recommend breaking this block of examples up and interspersing it with the text.